src/pages/index.js

```diff
@@ -27,6 +27,11 @@ export default function IndexPage({ data }) {
             <TopicCard category={cat} key={cat.name} />
           ))}
         </div>
+        <div className="text-center">
+          <Link to="/organizations" className="text-blue-400">
```

Comment:
I think the designs were a darker color
https://www.figma.com/file/GvYYLhGvEsLxS1T677r0Ox/Climatescape?node-id=504%3A1867

We dont have to be pixel-perfect, but something darker would be nice

Reply:
updated as well, tailwind didn't had that exact dark blue but i changed it to closest one, the other option is to pick it separately with inline styles.